### PR TITLE
[INLONG-1753][Feature][InLong-Manager] Add basic management APIs for Pulsar

### LINKED ISSUE #1753: [Feature][InLong-Manager] Add basic management APIs for Pulsar
### Description

Support the creation, reading, updates, and deletion of Pulsar related configuration.

### Use case

_No response_

### Are you willing to submit PR?

- [x] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by healchow**
### Title Name: [INLONG-1753][Feature][InLong-Manager] Add basic management APIs for Pulsar

Fixes #1753 

Related issue: #1334

### Motivation

Add basic management APIs for Pulsar.

### Modifications

Add business pulsar entity and operate APIs.

### Verifying this change

- [] Make sure that the change passes the CI checks.

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? yes
  - If yes, how is the feature documented? JavaDocs

### CODE REVIEW

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/beans/ClusterBean.java [hunk 1]
```diff
@@ -43,4 +43,10 @@
     @Value("${sort.appName}")
     private String appName;
 
+    @Value("${pulsar.admin}")
+    private String pulsarAdmin;
+
+    @Value("${pulsar.service}")
+    private String pulsarService;
```
Comment (aloyszhang) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/beans/ClusterBean.java:50]: It's better to rename these two parameters with `url` suffix like `pulsarAdminUrl` and `pulsarServiceUrl`

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/business/BusinessApproveInfo.java [hunk 2]
```diff
@@ -43,4 +43,34 @@
     @ApiModelProperty(value = "Data type name")
     private String schemaName;
 
+    @ApiModelProperty(value = "The partition num of Pulsar topic, between 1-20")
+    private Integer topicPartitionNum;
+
+    @ApiModelProperty(value = "Leader's number of writable nodes")
```
Comment (aloyszhang) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/business/BusinessApproveInfo.java:49]: Not leader, it's ledger

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/business/BusinessApproveInfo.java [hunk 3]
```diff
@@ -43,4 +43,34 @@
...
+    private Integer ensemble;
+
+    @ApiModelProperty(value = "Leader's number of copies")
```
Comment (aloyszhang) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/business/BusinessApproveInfo.java:52]: ditto

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/business/BusinessInfo.java [hunk 4]
```diff
@@ -52,12 +52,21 @@
...
     private String mqResourceObj;
 
+    @ApiModelProperty(value = "Tube master URL")
+    private String tubeMaster;
+
+    @ApiModelProperty(value = "Pulsar admin URL")
+    private String pulsarAdmin;
+
+    @ApiModelProperty(value = "Pulsar service URL")
+    private String pulsarService;
```
Comment (aloyszhang) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/business/BusinessInfo.java:68]: please add `url` suffix

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/datastream/DataStreamTopicVO.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
+package org.apache.inlong.manager.common.pojo.datastream;
```
Comment (aloyszhang) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/datastream/DataStreamTopicVO.java:18]: missing license header

**Comment by aloyszhang** [comment]
@healchow Overall LGTM, left some comments, PTAL

File: docker/docker-compose/sql/apache_inlong_manager.sql [hunk 6]
```diff
@@ -106,6 +106,28 @@ CREATE TABLE `business`
...
+    `ensemble`            int(3)      DEFAULT NULL COMMENT 'The writable nodes number of leader',
+    `write_quorum`        int(3)      DEFAULT NULL COMMENT 'The copies number of leader',
+    `ack_quorum`          int(3)      DEFAULT NULL COMMENT 'The number of requested acks',
+    `retention_time`      int(11)     DEFAULT NULL COMMENT 'Message storage time',
+    `retention_time_unit` char(20)    DEFAULT NULL COMMENT 'The unit of the message storage time',
+    `ttl`                 int(11)     DEFAULT NULL COMMENT 'Message time-to-live duration',
+    `ttl_unit`            varchar(20) DEFAULT NULL COMMENT 'The unit of time-to-live duration',
+    `retention_size`      int(11)     DEFAULT NULL COMMENT 'Message size',
+    `retention_size_unit` varchar(20) DEFAULT NULL COMMENT 'The unit of message size',
+    `is_deleted`          tinyint(1)  DEFAULT '0' COMMENT 'Whether to delete, 0: not deleted, 1: deleted',
```
Comment (inter12) [inline docker/docker-compose/sql/apache_inlong_manager.sql:126]: any table should have create_time and modify_time field。

File: docker/docker-compose/sql/apache_inlong_manager.sql [hunk 7]
```diff
@@ -106,6 +106,28 @@ CREATE TABLE `business`
...
+-- ----------------------------
+-- Table structure for business_pulsar
+-- ----------------------------
+DROP TABLE IF EXISTS `business_pulsar`;
+CREATE TABLE `business_pulsar`
+(
+    `id`                  int(11)      NOT NULL AUTO_INCREMENT COMMENT 'Incremental primary key',
+    `inlong_group_id`     varchar(128) NOT NULL COMMENT 'Business group id, filled in by the user, undeleted ones cannot be repeated',
+    `ensemble`            int(3)      DEFAULT NULL COMMENT 'The writable nodes number of leader',
+    `write_quorum`        int(3)      DEFAULT NULL COMMENT 'The copies number of leader',
```
Comment (inter12) [inline docker/docker-compose/sql/apache_inlong_manager.sql:118]: default value should not be null . it Unfriendly mysql index

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/DataStreamServiceImpl.java [hunk 8]
```diff
@@ -575,6 +576,16 @@ public int selectCountByGroupId(String groupId) {
         return count;
     }
 
+    @Override
+    public List<DataStreamTopicVO> getTopicList(String groupId) {
+        LOGGER.debug("begin bo get topic list by group id={}", groupId);
+        Preconditions.checkNotNull(groupId, BizConstant.GROUP_ID_IS_EMPTY);
+
+        List<DataStreamTopicVO> topicList = streamMapper.selectTopicList(groupId);
+        LOGGER.debug("success to get topic list");
```
Comment (inter12) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/DataStreamServiceImpl.java:585]: add more request info in debug is better。eg. groupid

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/business/BusinessApproveInfo.java (hunk 2 above)
Comment (healchow) [reply inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/business/BusinessApproveInfo.java]: fixed.

File: docker/docker-compose/sql/apache_inlong_manager.sql (hunk 6 above)
Comment (healchow) [reply docker/docker-compose/sql/apache_inlong_manager.sql]: add those two fields.

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/business/BusinessInfo.java (hunk 4 above)
Comment (healchow) [reply inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/business/BusinessInfo.java]: added.

File: docker/docker-compose/sql/apache_inlong_manager.sql (hunk 7 above)
Comment (healchow) [reply docker/docker-compose/sql/apache_inlong_manager.sql]: changed.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/DataStreamServiceImpl.java (hunk 8 above)
Comment (healchow) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/DataStreamServiceImpl.java]: added more info.

**Review by dockerzhang — APPROVED** [review]

**Review by aloyszhang — APPROVED** [review]
Great job!

**Pull request merged**